crates/rethnet_evm/benches/state.rs

```diff
@@ -45,11 +45,106 @@ fn bench_code_by_hash(c: &mut Criterion) {
     );
 }
 
+fn bench_database_commit(c: &mut Criterion) {
```

Comment:
Given that all 3 other traits have their own file (state, state_debug, and state_history), I feel like it would make sense to also have a db_commit file